.travis.yml

```diff
@@ -38,13 +38,13 @@ matrix:
            NUMPY_VERSION="1.10.4" SCIPY_VERSION="0.16.1" CYTHON_VERSION="0.25.2"
            PILLOW_VERSION="4.0.0" COVERAGE=true
       if: type != cron
-    # This environment tests the newest supported Anaconda release.
+    # This environment tests the newest supported Anaconda release (5.2.0).
```

Comment:
Minor: you can remove the version number here as we always take the latest version.